### PR TITLE
Fix unit tests when extensionrc and notificationrc files are used for config

### LINKED ISSUE #1057: Getting Error while running
Hi,
while running npm run test command in extension module I am getting error.Please suggest me on this.
payment

### PR DESCRIPTION
**Pull request by praveenkumarct**
Fixes #1057

### CODE REVIEW

File: extension/test/unit/config/config.spec.js [hunk 1]
```diff
@@ -786,4 +791,19 @@ describe('::config::', () => {
...
+    const currentFilePath = fileURLToPath(import.meta.url)
+    const currentDirPath = path.dirname(currentFilePath)
+    const projectRoot = path.resolve(currentDirPath, '../../../')
+    const pathToFile = path.resolve(projectRoot, '.extensionrc')
+
+    // Remove .extensionrc file if already exists
+    fs.stat(pathToFile, function (err, stats) {
+      //ignore error
+      if (!err) {
+        fs.unlinkSync(pathToFile)
```
Comment (leungkinghin) [inline extension/test/unit/config/config.spec.js:805]: Did you check if we still need to use `.extesionrc `after the the unit test running?

File: notification/test/unit/config.spec.js [hunk 2]
```diff
@@ -202,4 +205,19 @@ describe('::config::', () => {
...
+    const currentFilePath = fileURLToPath(import.meta.url)
+    const currentDirPath = path.dirname(currentFilePath)
+    const projectRoot = path.resolve(currentDirPath, '../../')
+    const pathToFile = path.resolve(projectRoot, '.notificationrc')
+
+    // Remove .notificationrc file if already exists
+    fs.stat(pathToFile, function (err, stats) {
+      //ignore error
+      if (!err) {
+        fs.unlinkSync(pathToFile)
```
Comment (leungkinghin) [inline notification/test/unit/config.spec.js:219]: Same as above

**Review by leungkinghin — COMMENTED** [review]
Looks good to me

File: extension/test/unit/config/config.spec.js (hunk 1 above)
Comment (praveenkumarct) [reply extension/test/unit/config/config.spec.js:805]: Same like config environment variable, We have to create .extensionrc file, If we want to start the application locally.

Comment (leungkinghin) [reply extension/test/unit/config/config.spec.js:805]: Does it mean if we unlink the file here, the application cannot retrieve the config from it?

Comment (praveenkumarct) [reply extension/test/unit/config/config.spec.js:805]: unlink is to remove the file. This file will be deleted if it exists already.
You have to recreate it again to run the application.

Comment (praveenkumarct) [reply extension/test/unit/config/config.spec.js]: https://github.com/commercetools/commercetools-adyen-integration/pull/1062/commits/78f8e0dff785f92f911b49eab00d5489d2612e74

I am temporarily renaming the file and naming it back later after the test.
Let me know if you have any better idea here.

**Review by leungkinghin — APPROVED** [review]
LGTM. Please fix the code style before merging.

**Pull request merged**